### ISSUE TITLE
I think maybe there's a simple mistake

### ISSUE THREAD

**Issue by sngo666** [ARM-software/astc-encoder#484]
in file [astc_blend_test.cpp and line 113](https://github.com/ARM-software/astc-encoder/blob/8bc51bc9bcd7620b1bc69a0f817a816af10509c6/Utils/astc_blend_test.cpp#L113) : 

Is it should be `use_filter = false;`? lol

**Comment by solidpixel** [comment]
Yes, it should indeed. We already have a PR queued to fix that one (#479).
